web/modules/custom/pineapple_helper/pineapple_helper.module

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * @file
+ * Contains pineapple_helper.module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use \Drupal\Core\Url;
+
+/**
+ * Implements hook_help().
+ */
+function pineapple_helper_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    // Main module help for the pineapple_helper module.
+    case 'help.page.pineapple_helper':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('A module allowing site users to report any site issues to the site administrator.') . '</p>';
+      return $output;
+
+    default:
+  }
+}
+
+/**
+ * Implements pineapple_helper_page_title_preprocessor().
+ */
+function pineapple_helper_preprocess_page_title(&$variables) {
+    $login_routes = array (
+      'user.login',
+      'user.register',
+      'user.pass',
+      'user.cancel_confirm',
+      'user.reset',
+      'user.reset.login',
+      'user.reset.form'
+    );
+
+    $route = \Drupal::routeMatch()->getRouteName();
+
+    // If the login_routes array contains the current route, return.
+    if (in_array($route, $login_routes)) {
+      return;
+    }
+
+    // If this is an admin page, or is an exception, return.
+    if (!\Drupal::service('router.admin_context')->isAdminRoute() || Drupal::request()->attributes->has('exception')) {
+      return;
+    }
+
+    // Create a form element of type container, which is a title suffix.
+    $variables['title_suffix']['help_form_spacer'] = array(
+      '#type' => 'container',
+      '#attributes' => array(
+        'class' => [
+          'pineapple-helper__spacer'
+        ],
+      ),
+      '#cache' => [
+        'contexts' => [
+          'route.name',
+        ],
+      ],
```

Comment:
Ready into drupal caching and include comment for this.

Reply:
done.